catalogs/catalog-lakehouse-paimon/src/main/java/com/datastrato/gravitino/catalog/lakehouse/paimon/PaimonCatalogOperations.java

```diff
@@ -98,7 +165,7 @@ public PaimonSchema loadSchema(NameIdentifier identifier) throws NoSuchSchemaExc
   @Override
   public PaimonSchema alterSchema(NameIdentifier identifier, SchemaChange... changes)
       throws NoSuchSchemaException {
-    throw new UnsupportedOperationException();
+    throw new UnsupportedOperationException("Alter schema is not supported in Paimon Catalog.");
```

Comment:
Paimon doesn't support alter schema or we doesn't support ?

Reply:
paimon

Reply:
https://github.com/apache/paimon/blob/master/paimon-core/src/main/java/org/apache/paimon/catalog/Catalog.java